File: vortexdbg-api/src/main/java/com/vortexdbg/unix/struct/TimeZone.java
package com.vortexdbg.unix.struct;

import com.vortexdbg.pointer.VortexdbgStructure;
import com.sun.jna.Pointer;

import java.util.Arrays;
import java.util.List;

public class TimeZone extends VortexdbgStructure {

    public TimeZone(Pointer p) {
        super(p);
    }

    public int tz_minuteswest;
    public int tz_dsttime;

    @Override
    protected List<String> getFieldOrder() {
        return Arrays.asList("tz_minuteswest", "tz_dsttime");
    }

}

File: vortexdbg-api/src/main/kotlin/com/vortexdbg/unix/struct/TimeZone.kt
package com.vortexdbg.unix.struct

import com.vortexdbg.pointer.VortexdbgStructure
import com.sun.jna.Pointer

class TimeZone(p: Pointer?) : VortexdbgStructure(p) {

    @JvmField
    var tz_minuteswest: Int = 0
    @JvmField
    var tz_dsttime: Int = 0

    override fun getFieldOrder(): List<String> {
        return listOf("tz_minuteswest", "tz_dsttime")
    }
}
